07.ls_object/outputter.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require_relative 'file_detail'
+
+class Outputter
+  attr_reader :file_names
+
+  COLUMNS = 3
+
+  def initialize(file_names)
+    @file_names = file_names
+  end
+
+  def output
+    formatted_names.each do |name|
```

Comment:
```suggestion
    formatted_names.each do |names|
```

配列であれば原則として複数形にした方がいいと思います